### ISSUE TITLE
Add / Link instruction to set up local repo clone

### ISSUE THREAD

**Issue by Stephen-X** [saberzero1/quartz-themes#18]
Hey there,

First off, many thanks for setting up this repo for Quartz themes! I'm liking it a lot so far, the better named css variables, the wonderful [Obsidian Minimal](https://github.com/kepano/obsidian-minimal) layout, etc.

This may be still in the early days so there's no rush, but do you think you can make a quick update to README with a link to [Setting up your GitHub repository - Quartz](https://quartz.jzhao.xyz/setting-up-your-GitHub-repository) for setting up the git submodule?

If people want to add new or make modifications to the themes, they'd need to follow that guide to set up their own theme repo and link that as a submodule, otherwise they'd be attempting to push changes to this repo instead.

**Comment by saberzero1** [comment]
Yeah, I'm first focussed on porting the entire Obsidian theme catalog.

https://github.com/saberzero1/quartz-themes/pull/16
